pkg/cgroups/cgroups.go

```diff
@@ -72,6 +73,30 @@ func Find() (Subsystems, error) {
 	return ret, nil
 }
 
+func FindWithParentGroup(podID string) (Subsystems, error) {
+	subsys, err := Find()
+	if err != nil {
+		return Subsystems{}, err
+	}
+
+	node, err := os.Hostname()
+	if err != nil {
+		return Subsystems{}, err
```

Comment:
i always try to wrap errors that come from other libraries with a `util.Errorf()` so we can easily tell where the error occurred

Reply:
but this doens't need to be a merge blocker